app/src/main/res/layout/empty_view_layout.xml

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-              xmlns:app="http://schemas.android.com/apk/res-auto"
-              android:layout_width="match_parent"
-              android:layout_height="match_parent"
-              android:orientation="vertical"
-              android:gravity="center"
-              android:layout_centerInParent="true"
-              android:paddingLeft="40dp"
-              android:paddingRight="40dp"
-              xmlns:tools="http://schemas.android.com/tools">
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:gravity="center"
+    android:layout_centerInParent="true"
+    android:paddingLeft="40dp"
+    android:paddingRight="40dp"
+    xmlns:tools="http://schemas.android.com/tools">

```

Comment:
Please do not reformat xml files. They are pretty inconsistent and I do not want to generate noise until we have decided for a consistent format that is enforced using checkstyle.

Reply:
That reformat was on accident (automatic). Will fix.